public/about.html

```diff
@@ -33,4 +33,11 @@
       </div>
     </div>
   </body>
+  <footer>
+    <div class="container giphy">
```

Comment:
This `<div>` and classes seem unnecessary. Is that indeed the case or am I missing something?